ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventGAspec.json

```diff
@@ -118,5 +118,24 @@
         "CRUD": "CRU"
       }
     ]
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseEventID": "JUDGE_MAKES_DECISION",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-solicitor"
+        ],
+	    	"CRUD": "CRU"
```

Comment:
The CRU permissions for solicitor will be changed to R once the role assignment for JUDGE is sorted out.